### ISSUE TITLE
请问accuracy的形参是不是有误？

### ISSUE THREAD

**Issue by LJXLJXLJX** [jinze1994/DeepID1#2]
deepid1.py 中
def accuracy(y_estimate, y_real):
    with tf.name_scope('accuracy'):
        with tf.name_scope('correct_prediction'):  
            correct_prediction = tf.equal(tf.argmax(y,1), tf.argmax(y_,1))
        with tf.name_scope('accuracy'):  
            accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
        tf.scalar_summary('accuracy', accuracy)  
        return accuracy

形参y_estimate, y_real  好像在函数体中没有出现

**Comment by weihualiuhupituzi** [comment]
作者应该是笔误
